tests/unit/test_table.py

```diff
@@ -1455,12 +1540,15 @@ def _make_responses(self, codes):
         return MutateRowsResponse(entries=entries)
 
     def test_callable_empty_rows(self):
-        from google.cloud.bigtable_v2.gapic import bigtable_client
-        from google.cloud.bigtable_admin_v2.gapic import bigtable_table_admin_client
+        from google.cloud.bigtable_v2.services.bigtable import BigtableClient
+        from google.cloud.bigtable_admin_v2.services.bigtable_table_admin import (
+            client as bigtable_table_admin,
+        )
 
-        data_api = mock.create_autospec(bigtable_client.BigtableClient)
-        table_api = mock.create_autospec(
-            bigtable_table_admin_client.BigtableTableAdminClient
+        data_api = mock.create_autospec(BigtableClient)
+        table_api = mock.create_autospec(bigtable_table_admin.BigtableTableAdminClient)
+        table_api.table_path.return_value = (
+            "projects/self.PROJECT_ID/instances/self.INSTANCE_ID/tables/self.TABLE_ID"
```

Comment:
is the project name literally `self.PROJECT_ID`?

Reply:
Hahaha, this is such a good catch. This was unnecessary - deleted :)